receipt_agent/receipt_agent/agents/label_evaluator/financial_structured.py

```
@@ -0,0 +1,809 @@
+
+
+def _run_two_tier_core(
+    words,
+    labels,
+    visual_lines,
+    chroma_client,
+    image_id: str,
+    receipt_id: int,
+    merchant_name: str,
+    *,
+    skip_fast_path: bool,
```

Ruff (0.15.13)

[warning] Unused function argument: `skip_fast_path`

(ARG001)